### PR TITLE
Firebase error

### PR DESCRIPTION
**Pull request by ReactwithHemanth**
compatibility screen - details screen
footer visibility setup

### CODE REVIEW

File: src/Confg/Firebase.js
Comment (vinay-sayone) [inline src/Confg/Firebase.js:1]: Do we need this config file if we are using the react native firebase plugin. These are already getting configured just by adding the GoogleService json/ GoogleInfoPlist file right ?

File: src/Screens/EntryScreens/NumberSignUp.js [hunk 1]
```diff
@@ -76,6 +76,27 @@ const NumberSignUp = () => {
             <PhoneInput
               value={phoneNumber}
               defaultCountry="IN"
+              phoneInputStyles={{
+                container: {borderWidth: 0, backgroundColor: 'transparent'},
+                input: {
+                  width: '70%',
+                  height: 45,
+                  backgroundColor: '#fff',
+                  margin: 5,
```
Comment (vinay-sayone) [inline src/Screens/EntryScreens/NumberSignUp.js]: Remove inline styles and move them to a separate stylesheet

File: src/Screens/Footer/CompatibilityScreen.js [hunk 2]
```diff
@@ -1,46 +1,215 @@
...
+    ref.current?.scrollToIndex({
+      index: index == 8 ? 0 : index,
+      animated: true,
+      viewOffset: 0.5 || 1 ? 0 : SPACING,
+    });
+
+    // ref2.current?.scrollToIndex({
+    //   index: Myindex,
+    //   animated: true,
+    //   viewOffset: 0.5 || 1 ? 0 : SPACING,
```
Comment (vinay-sayone) [inline src/Screens/Footer/CompatibilityScreen.js]: Remove comments if not needed

**Review by vinay-sayone — CHANGES_REQUESTED** [review]
Remove inline styles if any. Also remove unwanted comments / console logs.

File: src/Confg/Firebase.js
Comment (ReactwithHemanth) [reply src/Confg/Firebase.js:1]: that was part of previous codebase which is unstable and its is not, and removed by latest codebase, that will be pushing

File: src/Screens/EntryScreens/NumberSignUp.js (hunk 1 above)
Comment (ReactwithHemanth) [reply src/Screens/EntryScreens/NumberSignUp.js]: Rectified

File: src/Screens/Footer/CompatibilityScreen.js (hunk 2 above)
Comment (ReactwithHemanth) [reply src/Screens/Footer/CompatibilityScreen.js]: removed

File: src/Confg/Firebase.js
Comment (ReactwithHemanth) [reply src/Confg/Firebase.js:1]: Removed the config file

**Review by vinay-sayone — APPROVED** [review]
Approved

**Pull request merged**